main.js

```diff
@@ -194,3 +213,13 @@ $catalogView.addEventListener('click', (event) => {
   }
 
 })
+
+$detailsView.addEventListener('click', (event) => {
+  var $cartBtn = document.querySelector('#add-to-cart')
```

Comment:
The `event.target` might have an `id` attribute of `'add-to-cart'`, right? No need to reach into the page.